### PR TITLE
17 update instructions for development

### LINKED ISSUE #17: Update instructions for development
The [Development](https://agilepy.readthedocs.io/en/1.6.3/help/development.html) page of Documentation mention the possibility to develop in conda environment. Alessandro ursi tried but failed due to lack of environment.yml.

Also the Docker section mentions conda inside Docker, wasn't it removed in v1.6.0?

1. We need to update the Documentation page with one clear set of instructions.
2. @LeoBaro When we need to update a package version and test the existing code in a new environment what should we change?

@AndreaBulgarelli

### PR DESCRIPTION
**Pull request by gabriele-panebianco-inaf**
Update development instructions. Close #17 

In _docs/source/help/development.rst_ 

1.  Remove references to development with Anaconda. Development with anaconda used to be done with
    ```
    conda create -n agilepydev -c agilescience agiletools agilepy-dataset
    conda activate agilepydev
    conda env update -f environment.yml
    python setup.py develop
    ```
    I left this code as a comment inside development.rst if we ever need it again.

2. Update the Docker Development installation with step-by-step instructions.

Also update references to development in _docs/source/quickstart/installation.rst_ accordingly.

**IMPORTANT**

- [ ] In setup.py change agilepy version number 1.6.2 -> 1.6.3. I reckon we forgot to update it. Is it correct or should we change it back?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 setup( name='agilepy',
-       version='1.6.2',
+       version='1.6.3',
        author='Baroncelli Leonardo, Addis Antonio, Bulgarelli Andrea, Parmiggiani Nicolò, Ambra Di Piano, Gabriele Panebianc',
```
Comment (LeoBaro) [inline setup.py:7]: A 'o' is missing in "Gabriele Panebianc"

**Review by LeoBaro — CHANGES_REQUESTED** [review]
Great work! Only two comments to address. Thanks!

File: docs/source/help/development.rst [hunk 2]
```diff
@@ -2,50 +2,107 @@
...
-    python setup.py develop
-
-Docker
-------
+Instructions
+------------
 
-::
+1. Prepare your workspace by creating a directory :code:`agile` at
+path :code:`$PATH_TO_AGILE`.
```
[the remark this answers is not in the document]
Comment (gabriele-panebianco-inaf) [reply docs/source/help/development.rst]: Thanks @LeoBaro 
The user has to define it; it is the path to folder `agile` on the user's local host.
I pushed a commit where we say that explicitly (instructions, step 1) and add the following line
```
$ export PATH_TO_AGILE=$PWD
```

File: setup.py (hunk 1 above)
Comment (gabriele-panebianco-inaf) [reply setup.py]: Thanks, fixed in the latest commit

**Review by LeoBaro — APPROVED** [review]

**Pull request merged**